src/apple/apple-wizard.ts

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable max-lines */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import clack from '@clack/prompts';
+import chalk from 'chalk';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as templates from './templates';
+import * as xcManager from './xcode-manager';
+import * as codeTools from './code-tools';
+import * as bash from '../utils/bash';
+import { WizardOptions } from '../utils/types';
+
+const xcode = require('xcode');
+/* eslint-enable @typescript-eslint/no-unused-vars */
+
+import {
+  askForProjectSelection,
+  askForSelfHosted,
+  askForWizardLogin,
+  askToInstallSentryCLI,
+  SentryProjectData,
+  printWelcome,
+} from '../utils/clack-utils';
+
+export async function runAppleWizard(
+  options: WizardOptions,
+): Promise<void> {
+  printWelcome({
+    wizardName: 'Sentry Apple Wizard',
+    promoCode: options.promoCode,
+  });
+
+
+  if (!bash.hasSentryCLI()) {
+    if (!(await askToInstallSentryCLI())) {
+      clack.log.warn("Without Sentry-cli, you won't be able to upload debug symbols to Sentry. You can install it later by following the instructions at https://docs.sentry.io/cli/");
+    } else {
+      await bash.installSentryCLI();
+    }
+  }
+
+  const projectDir = process.cwd();
+  const xcodeProjFile = findFilesWithExtension(projectDir, ".xcodeproj")[0];
+
+  if (!xcodeProjFile) {
+    clack.log.error('No xcode project found. Please run this command from the root of your project.');
+    return;
+  }
+
+  const pbxproj = path.join(projectDir, xcodeProjFile, "project.pbxproj");
```

Comment:
Just to sanity-check (for someone who has no idea about xcode/ios apps): Is this a valid path? 

```
wizard/wizard-apple/test.xcodeproj/project.pbxproj
```

Background: I briefly tried the wizard locally but couldn't get beyond the pbxproj step:

![image](https://github.com/getsentry/sentry-wizard/assets/8420481/a7200c6b-6a31-42bd-a793-f1980f1a958e)

Reply:
Yes. The ".xcodeproj" is actually a folder that keeps project configuration files.

I think we should not show the "Successfully..." message in this case.

> I briefly tried the wizard locally but couldn't get beyond the pbxproj step:

You just tried to run it, or you really have an xcode project to test it?

Reply:
> Yes. The ".xcodeproj" is actually a folder that keeps project configuration files.

 Okay, all good then! I guess I was just confused by the `xcodeProjectFile` variable then 😅 

> You just tried to run it

Yeah, but not very methodically and without an actual xcode project. Just wanted to get over a few steps so I created some empty files 😅

Reply:
@brustolin do you know a repo which would be a example app to test this stuff with ourselves?

Reply:
I do 😁 https://github.com/brustolin/XcodeProjects